Q/QD/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+using BinaryBuilder
+
+name = "QD"
+version = v"2.3.22"
+
+# Collection of sources required to build SDPA-QD
+sources = [
+    "https://www.davidhbailey.com/dhbsoftware/qd-2.3.22.tar.gz" =>
+    "30c1ffe46b95a0e9fa91085949ee5fca85f97ff7b41cd5fe79f79bab730206d3",
+    "./bundled",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/qd-2.3.22
+update_configure_scripts
+./configure --enable-shared --enable-fast-install=no  --prefix=$prefix --host=$target --build=x86_64-linux-gnu
```

Comment:
The build environment is `x86_64-linux-musl`, but you don't need to remember it, you can use the `MACHTYPE` environment variable:
```suggestion
./configure --enable-shared --enable-fast-install=no  --prefix=$prefix --host=$target --build=${MACHTYPE}
```